### PR TITLE
Resolve custom processor

### LINKED ISSUE #1040: Custom flavour extensions should be in the npm package
And before trying to load an external `js` file with `--customFlavour`, it should be checked in the internal `extensions` directory.

So:
* `--customFlavour=WiktionaryFR` should load `WikitionaryFR.js` in this extensions directory.
* `--customFlavour=foobar` should load `foobar.js` in the current directory (from where `mwoffliner` is launched by the user).
* `--customFlavour=foobar.js` should load `foobar.js` in the current directory (from where `mwoffliner` is launched by the user).
* `--customFlavour=/foobar.js` should load `foobar.js` in the root directory directory

### PR DESCRIPTION
**Pull request by LakmaNeha**
Resolves #1040

### CODE REVIEW

**Comment by LakmaNeha** [comment]
@kelson42 I am not sure why tests are failing https://travis-ci.com/github/openzim/mwoffliner/jobs/298122414. I can't find anything that would affect the failing test cases, can you help me figure out this. Thanks.

File: src/mwoffliner.lib.ts [hunk 1]
```diff
@@ -25,7 +25,7 @@ import Redis from './Redis';
 import { writeFilePromise, mkdirPromise, isValidEmail, genHeaderCSSLink, genHeaderScript, genCanonicalLink, saveStaticFiles, readFilePromise, makeArticleImageTile, makeArticleListItem, getDumps, getMediaBase, MIN_IMAGE_THRESHOLD_ARTICLELIST_PAGE, downloadAndSaveModule, getSizeFromUrl, getRelativeFilePath } from './util';
 import { mapLimit } from 'promiso';
 import logger from './Logger';
-import { getAndProcessStylesheets } from './util';
+import { getAndProcessStylesheets, getCustomProcessorPath } from './util';
```
Comment (kelson42) [inline src/mwoffliner.lib.ts]: Naming is wrong, you mean `getCustomFlavourPath`? secure the naming of variable is proper everywhere in your PR.

**Comment by kelson42** [comment]
> @kelson42 I am not sure why tests are failing https://travis-ci.com/github/openzim/mwoffliner/jobs/298122414. I can't find anything that would affect the failing test cases, can you help me figure out this. Thanks.

There is currently a problem with the CI, please ignore it.

**Comment by LakmaNeha** [comment]
@kelson42 resolved review comments. Please review

**Comment by kelson42** [comment]
No feedback, I close the ticket.

**Pull request closed without merging**